src/System.IO.FileSystem.AccessControl/src/Resources/Strings.resx

```diff
@@ -188,4 +188,16 @@
   <data name="Arg_PathEmpty" xml:space="preserve">
     <value>The path is empty.</value>
   </data>
+  <data name="ArgumentOutOfRange_NeedPosNum" xml:space="preserve">
+    <value>Positive number required.</value>
+  </data>
+  <data name="Argument_InvalidAppendMode" xml:space="preserve">
+    <value>Append access can be requested only in write-only mode.</value>
+  </data>
+  <data name="Argument_InvalidFileModeAndAccessCombo" xml:space="preserve">
+    <value>Combining FileMode: {0} with FileAccess: {1} is invalid.</value>
```

Comment:
Nit: Personally I think the colons look strange as part of a sentence. Consider
```
Combining FileMode.{0} with FileAccess.{1} is invalid.
```
or
```
Combining FileMode {0} with FileAccess {1} is invalid.
```

Reply:
Done.